shell/app/manifests.cc

```diff
@@ -22,3 +26,19 @@ const service_manager::Manifest& GetElectronContentBrowserOverlayManifest() {
           .Build()};
   return *manifest;
 }
+
+#if BUILDFLAG(ENABLE_BUILTIN_SPELLCHECKER)
+const service_manager::Manifest& GetElectronContentRendererOverlayManifest() {
+  static base::NoDestructor<service_manager::Manifest> manifest {
+    service_manager::ManifestBuilder()
+#if defined(OS_MACOSX)
+        .ExposeInterfaceFilterCapability_Deprecated(
+            "navigation:frame", "browser",
+            service_manager::Manifest::InterfaceList<
+                spellcheck::mojom::SpellCheckPanel>())
```

Comment:
After looking through , I don't see the `SpellCheckPanel` initialized anywhere in the renderer , can you remove this dependency if not needed ?

Reply:
Without it some internal logic crashes.  I think it _can_ get triggered from a menu item role on macOS.

Reply:
Thats weird, this just provides access to the service interface for the renderer process and I don't see any use of this interface in here. Can you track down where it is coming from ?

Reply:
I'll give that another look